src/Middleware/RedirectToLogin.php

```diff
@@ -29,19 +30,22 @@
  */
 class RedirectToLogin implements MiddlewareInterface
 {
+    public UrlStore $UrlStore; //added this
```

Comment:
should be private or protected. First letter of Properties/Variables should always be lowercase.